server/src/modules/user/user.interface.ts

```diff
@@ -1,9 +1,13 @@
 export interface Iconfiguration {
-  port: number;
-  database: string;
-  jwtSecret: string;
-  tokenExpiry: string | number;
-  refreshTokenExpiry: string | number;
+  PORT: number;
+  MONGODB_URL: string;
+  JWT_SECRET: string;
+  TOKEN_EXPIRY: string | number;
+  REFRESH_TOKEN_EXPIRY: string | number;
```

Comment:
this both should be come from constant file not from env variables

Reply:
updated